### PR TITLE
checker: infoschema does not need privilege to access

### LINKED ISSUE #7317: task precheck should not require SELECT privilege on INFORMATION_SCHEMA
### What did you do?

because INFORMATION_SCHEMA does not require privilege
img_v2_544ed6b3-0407-430d-9d17-54fe4bdd696g

So when only grant SELECT on one database to user

### What did you expect to see?

_No response_

### What did you see instead?

image

### Versions of the cluster

master

### current status of DM cluster (execute `query-status <task-name>` in dmctl)

```console
(paste current status of DM cluster here)
```

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: close #7317 

### What is changed and how it works?

so no need to let checker require this privilege

#### Tests 

 - Unit test

### Release note 

```release-note
DM precheck no longer reports lacking privileges of INFORMATION_SCHEMA
```

### CODE REVIEW

File: dm/pkg/checker/privilege_test.go [hunk 1]
```diff
@@ -65,7 +68,7 @@ func TestVerifyDumpPrivileges(t *testing.T) {
 			},
 			// `db1`.`anomaly_score`; `INFORMATION_SCHEMA`
```
Comment (okJiang) [inline dm/pkg/checker/privilege_test.go:69]:
```suggestion
			// `db1`.`anomaly_score`
```

**Comment by okJiang** [comment]
should we cherry-pick to 6.1?

**Review by okJiang — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**